### ISSUE TITLE
Standar deviation of list to list

### ISSUE THREAD

**Issue by AlexisABG** [binary-com/binary-bot#2953]
Hi folks, I need your help.

I'm trying to make a list of standard deviation value from a list block, but it rise a jquery error, how can I make a list of this value?
Neither is possible to assign the block result to a variable because it raise a jquery error too.

Any idea how to solve it? 
![bot error](https://user-images.githubusercontent.com/68515474/90026420-e4266b80-dc7c-11ea-9fa4-d199e8417d87.png)
@raunakkathuria @Mahboobeh-binary @ashkanx @aminmarashi-binary @shudarshon-binary

**Comment by aaron-binary** [comment]
Hey @AlexisABG,

It seems the `STD15` variable is not initialised? You can add a `Set STD15 to: empty list` in the `Run once at start` statement.

Regards,
Aaron

**Comment by AlexisABG** [comment]
Hey @aaron-binary  I've initialized an empty list, I've tried to assign the block to a variable even I've tried to create a function that returns the standard deviaton of the list as a unique value but it seems nothing works.

For example in the image above I try to assign the standard deviation to a variable named STD15 but it does not work.

Thanks in advance for any solution that you or the community can offer to me.
